### ISSUE TITLE
Improve human readability of saved files

### ISSUE THREAD

**Issue by beeedy** [CamelCaseNotation/api-ai-git#1]
While the current pickle format works it seems that a more human readable format would be beneficial so that by diff-ing two files you could easily get an idea of what was changed. Personally I think a stretch goal would be to back up into the format found within api.ai's .zip file they offer for manual backups (no idea why there isn't access to the backup/restore feature through their api)

**Comment by texascloud** [comment]
Good idea! I was using pickle files to try to save space in the event someone's agent had a large amount of intents and entities, but it's likely pre-mature optimization. I'll try to have this done by next week as I'm currently in the process of moving, thanks!

**Comment by beeedy** [comment]
I verified that you can remove the agent.json and customDomainsResponses.json from the backup .zip file and the built in restore functionality within API.ai will still accept the .zip as a valid backup. It may be beneficial to go with this structuring so that this tool isn't *required* to restore an agent, plus it is pretty human readable!
